app/controllers/dashboards_controller.rb

```diff
@@ -3,7 +3,6 @@ class DashboardsController < BaseController
   before_action :set_policy_with_context, except: [:refresh_dashboard]
   before_action :set_dashboard
   before_action :set_defence_requests, only: [:show, :closed]
-  before_action :set_dashboard_count, only: [:show, :closed]
```

Comment:
Yeah, I'm sure that the policy check could work with the `id` automatically. I'm not too familiar with the policy implementation, but the `id` could be considered a scope and the policy context would automatically verify that the user (and type) is authorised.